tests/acls/acl.sh.in

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
- Put a single bind to line on netdata.conf before the tests start, with many different ports and settings, including SSL. You can replace it with the original value at the end. This will also test that multiple configurations are passed properly, not just one config for one port.
- Pass the port and the protocol http/https to run_acl_tests and add tests for SSL. 
- Add a simple API call that is not health management (also tests the dashboard). e.g. api/v1/info

Can you also test streaming this way? I don't think so, but maybe you can think of something.

Reply:
The health was added due the flag WEB_CLIENT_ACL_MGMT that was created for it, but considering that we have a specific test for health, I will remove this from the script.

Reply:
No, I meant leave the check for the health mgt api but also add another call.

Reply:
All right, it is almost done!